search/materialize.go

```diff
@@ -379,11 +388,16 @@ func (m *Materializer) materializeColumn(ctx context.Context, file *storage.Parq
 		return nil, errors.Wrap(err, "failed to materialize columns")
 	}
 
-	values := make([]parquet.Value, 0, len(rr))
-	for _, v := range rr {
-		values = append(values, r[v]...)
```

Comment:
because of the change above with the intersection, "v" is not a key of "r" anymore necessarily